crates/router/src/connector/paypal.rs

```diff
@@ -1044,6 +1045,17 @@ impl api::IncomingWebhook for Paypal {
                     api_models::webhooks::RefundIdType::ConnectorRefundId(resource.id),
                 ))
             }
+            paypal::PaypalResource::PaypalDisputeWebhooks(resource) => {
+                Ok(api_models::webhooks::ObjectReferenceId::PaymentId(
+                    api_models::payments::PaymentIdType::PaymentAttemptId(
+                        resource
+                            .dispute_transactions
+                            .first()
```

Comment:
how are we going to handle multiple dispute transactions here?

Reply:
@swangi-kumari  please confirm with connector, in which case do we get multiple transaction items for a dispute